pegasus/sites.v3/code.org/public/index2.haml

```diff
@@ -89,15 +106,17 @@ social:
               .caption3{style: "left: 60px; top: 156px; position: absolute; opacity: 0; font-size: 14px; background-color: rgba(0,0,0,0.5); padding:8px"}
                 = lines[2]
 
-      #leftvideo{style: "width: 100%"}
-        = view :index_video_reveal
-        %div{style:"clear:both"}
-        %br/
+      &nbsp;
+
+      #blankspaceleft{style: "text-align:right"}
+        %div{style: "width:100%; height: 100%; margin: 0 auto;"}
+          #videodiv{style: "position:relative;"}
+            %img{style: "display:block; width:100%; height:auto", src: "/images/16x9.png"}/        
```

Comment:
I'll add a comment.  It basically keeps the hero picture big, and for now uses a 16x9 rectangle which is how hourofcode.com also maintains room for the video player when it appears.  This might change, but right now it keeps layout sane.